src/test/java/com/junwoo/ott/domain/video/service/VideoServiceTest.java

```diff
@@ -0,0 +1,72 @@
+
+package com.junwoo.ott.domain.video.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.junwoo.ott.domain.video.common.VideoTestCommons;
+import com.junwoo.ott.domain.video.dto.response.VideoCreateResponseDto;
+import com.junwoo.ott.domain.video.entity.Video;
+import com.junwoo.ott.domain.video.repository.VideoJpaRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class VideoServiceTest implements VideoTestCommons {
+
+  @Mock
+  private VideoJpaRepository videoJpaRepository;
+
+  @InjectMocks
+  private VideoService videoService;
+
+  @Nested
+  @DisplayName("비디오 생성 성공")
+  class CreateVideo {
+
+    @Test
+    @DisplayName("생성 성공")
+    void 비디오생성() {
```

Comment:
실패는 명확하게 실패라고 붙여주셨으므로 성공도 성공으로 붙여주시면 좋을 것 같습니다
```suggestion
    void 비디오생성성공() {
```

Reply:
넵!